bin/saya/src/args/mod.rs

```diff
@@ -133,27 +128,38 @@ impl TryFrom<SayaArgs> for SayaConfig {
                 None => None,
             };
 
-            let prover_key = ProverAccessKey::from_hex_string(&args.private_key).map_err(|e| {
-                Box::new(std::io::Error::new(std::io::ErrorKind::InvalidInput, e.to_string()))
-            })?;
+            let starknet_account = StarknetAccountData {
```

Comment:
Let's use more rusty way of having some conversions using `TryFrom` or `From` if it can't fail.